src/tests/ftest/performance/performance_test_base.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env python3
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+import time
+import types
+
+from ior_test_base import IorTestBase
+from mdtest_test_base import MdtestBase
+#from mdtest_utils import MdtestMetrics
```

Comment:
mdtest will be used soon. I didn't want to remove MdtestBase, since that affects some of the instance methods and variables.